tests/test_enterprise/api/test_views.py

```diff
@@ -241,6 +213,21 @@ def test_user_view(self, factory, url, sorting_key, model_items, expected_json):
                 'consent_granted': True,
                 'course_id': 'course-v1:edX+DemoX+DemoCourse',
             }],
+        ),
+        (
+            factories.EnterpriseCustomerCatalogFactory,
+            reverse('enterprise-catalog-list'),
```

Comment:
`ENTERPRISE_CUSTOMER_CATALOG_LIST_ENDPOINT` is not the same endpoint as this -- I actually tripped up here and forgot to change the model serialization endpoint to `enterprise-customer-catalog-list` :) No one caught it earlier but, alas!